wagtail/admin/views/reports/aging_pages.py

```diff
@@ -52,24 +54,44 @@ class AgingPagesView(PageReportView):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+        self.user_model = get_user_model()
         self.custom_field_preprocess = self.custom_field_preprocess.copy()
         self.custom_field_preprocess["content_type"] = {
             self.FORMAT_CSV: get_content_type_label,
             self.FORMAT_XLSX: get_content_type_label,
         }
 
+    def user_id_to_python(self, user_id):
+        pk_field_class = type(self.user_model._meta.pk)
+        return pk_field_class().to_python(user_id)
```

Comment:
I've also tested a bit around trying different things, at the end your solutions seems to be the only one working :+1: 
Did you have any issues with that, or is there a reason why you don't use the existing field instance directly?
```python
return self.user_model._meta.pk.to_python(user_id)
```
There may be custom field classes, which require some passed attributes to work, we would avoid those issues.

Reply:
I wasn't sure about this actually - thanks for pointing it out.